### DISCUSSION TITLE
Applying Updates of the Plugin Across Projects

### DISCUSSION THREAD

**Discussion by kingargyle** [slackhq/foundry#11] [Q&A] [answered]
Within a large company, how are you guys handling the upgrade process for when a new version of the plugin is released.

Typically I've seen the use of + on the version numbers of the plugins, but this in itself can be dangerous as new versions released could be consumed to early.   Do you guys coordinate the updates, once a quarter or do you have scheduled release cycles?

**Comment by ZacSweers** [comment, marked as answer]
I think your question might be assuming this is done across multiple repos. We only use this plugin in one (currently) repo so updates are preeeeeetty easy! Essentially we just cut a new version and bump it in the consuming repo. If it's a more gnarly change, we'll test it a bit first via included build locally first.
